webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -484,7 +514,7 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     this.globalActions.setLeftActionBar({
       exportFn: () => this.exportContacts(),
       hasResults: this.hasContacts,
-      userFacilityId: this.usersHomePlace?._id,
+      userFacilityId: this.usersHomePlace?.[0]?._id,
```

Comment:
I'm seeing that this `this.usersHomePlace?.[0]?._id` is used in multiple places. 
While I accept that this is for the MVP, I think we should have it in a function and show that this is intentional, and reuse that function throughout.